app/views/layouts/application.html.slim

```diff
@@ -1,67 +1,81 @@
-                    = link_to user_messages_path(current_user) do
-                      = "Messages"
+      header.py-3
+        nav.navbar.navbar-expand-lg.fixed-top.bg-light.text-bg-light
+          .container
+            = link_to home_index_path, class: "navbar-brand"
+              | CodeHarbor
+            button.navbar-toggler[type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"]
+              span.navbar-toggler-icon
+            #navbarSupportedContent.collapse.navbar-collapse
+              - if user_signed_in?
+                ul.navbar-nav.me-auto.mb-2.mb-lg-0
+                  li.nav-item
+                    = link_to tasks_path, class: "nav-link #{active_class(tasks_path)}"  do
+                      i.fa.fa-file-text-o style=("color: #006600")
+                      = ' ' + t('layouts.tasks')
+                  li.nav-item
+                    = link_to collections_path, class: "nav-link #{active_class(collections_path)}"  do
+                      i.fa.fa-folder-open style=("color: #000066")
+                      = ' ' + t('layouts.collections')
+                  li.nav-item
+                    = link_to groups_path, class: "nav-link #{active_class(groups_path)}"  do
+                      i.fa.fa-group style=("color: #660000")
+                      = ' ' + t('layouts.groups')
+                  - if current_user.role === 'admin'
+                    li.nav-item
+                      = link_to rails_admin.dashboard_path, 'data-turbolinks': false, class:"nav-link" do
+                        i.fa.fa-lock
+                        = ' ' + t('layouts.admin_nav.rails_admin')
+              ul.navbar-nav.ms-auto
+                - if user_signed_in?
+                  li.nav-item.dropdown
+                    a.nav-link.dropdown-toggle href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false"
+                      = current_user.email
                       = ' '
                       - if current_user.unread_messages_count != '0'
                         span.menu-number
                           = current_user.unread_messages_count
-              = menu_item t('layouts.logout'), destroy_user_session_path, method: :delete
-            - else
-              = menu_item new_user_registration_path do
-                i.fa.fa-user
-                = ' ' + t('home.sign_up')
-              = menu_item new_user_session_path do
-                i.fa.fa-sign-in
-                = ' ' + t('home.login')
+                    ul.dropdown-menu aria-labelledby="navbarDropdown"
+                      li
+                        = link_to 'Profile', user_path(current_user), class: "dropdown-item"
+                      li
+                        = link_to user_messages_path(current_user), class:"dropdown-item" do
+                          = "Messages"
+                          = ' '
```

Comment:
This should also be moved to a translation and be simplified with:

```suggestion
                          => "Messages"
```

Similar to my comment above, this removes the single space string. Please change it globally as well.